src/main/java/qupath/ext/instanseg/core/InstanSeg.java

```diff
@@ -57,11 +59,19 @@ public class InstanSeg {
     private final Device device;
     private final TaskRunner taskRunner;
     private final Class<? extends PathObject> preferredOutputClass;
+    private final Map<String, Object> optionalArgs;
 
     // This was previously an adjustable parameter, but it's now fixed at 1 because we handle overlaps differently.
     // However we might want to reinstate it, possibly as a proportion of the padding amount.
     private final int boundaryThreshold = 1;
 
+    /**
```

Comment:
I'm confused why this has been moved up in the file, above the constructor and further from the other `detectObjects` methods.

Reply:
Me too